docs/examples/github-actions-cache.md

```diff
@@ -0,0 +1,46 @@
+# Example: GitHub Actions cache
+
+Jobs on Actuated runners start in a clean VM each time. This means dependencies need to be downloaded and build artifacts or caches rebuilt each time. Caching these files in the actions cache can improve workflow execution time.
+
+A lot of the setup actions for package managers have support for caching built-in. See: [setup-node](https://github.com/actions/setup-node#caching-global-packages-data), [setup-python](https://github.com/actions/setup-python#caching-packages-dependencies),... . They require minimal configuration and will create and restore dependency caches for you.
```

Comment:
Is there support for setup-rust? @Addyvan was asking about that

Reply:
There is no official setup-rust action. Not sure if the actions from [actions-rs](https://github.com/actions-rs) support it.

There is [an example](https://github.com/actions/cache/blob/main/examples.md#rust---cargo) to configure caching with rust available in the [actions/cache](https://github.com/actions/cache) repo.